platform/src/main/java/platform/pageobjects/Header/HeaderAfterLogin.java

```diff
@@ -0,0 +1,137 @@
+
+	@FindBy(xpath = "//div[@class='submenu']//ul[1]//li[1]//a[1]")
+	public WebElement accountAndSecuritySubmenuLink;
+
+	@FindBy(xpath = "//div[@class='submenu']//ul[1]//li[2]//a[1]")
+	public WebElement membershipSubMenuLink;
+
+	@FindBy(xpath = "//div[@class='submenu']//ul[1]//li[3]//a[1]")
+	public WebElement servicesYouAreUsingSubMenuLink;
+
+	@FindBy(xpath = "//div[@class='submenu']//ul[1]//li[4]//a[1]")
+	public WebElement orderHistorySubMenuLink;
+
+	public void switchToPreviousTab() {
+		ArrayList<String> tab = new ArrayList<>(driver.getWindowHandles());
+		driver.switchTo().window(tab.get(1));
+		driver.close();
+		driver.switchTo().window(tab.get(0));
+	}
+
+	public void clickSubMenu(WebElement x, WebElement y) throws Exception {
+
+		Actions actions = new Actions(driver);
+		CommonFunctions.waitForVisiblity(x, waitTime);
+		actions.moveToElement(x).perform();
+		log.info("Opened Submenu");
+		CommonFunctions.isElementVisible(y);
+		CommonFunctions.waitForVisiblity(y, waitTime);
+		y.click();
+		driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+	}
+
+	public void checkHeaderMenuAndSubMenuLinksVisibilityAfterLogin() throws Exception {
+
+		Actions actions = new Actions(driver);
+
+		actions.moveToElement(accountAndSecurityLink).perform();
+		CommonFunctions.isElementVisible(accountAndSecurityLink);
+		log.info("Menu : Account Security Link is visible");
+
+		actions.moveToElement(membershipButton).perform();
+		CommonFunctions.isElementVisible(membershipButton);
+		log.info("Menu : Membership Button Link is visible");
+
+		actions.moveToElement(accountAndSecurityLink).perform();
+		log.info("Opened Submenu");
+		CommonFunctions.isElementVisible(accountAndSecuritySubmenuLink);
+		log.info("SubMenu : AccountAndSecurity Link is visible");
+
+		actions.moveToElement(accountAndSecurityLink).perform();
+		CommonFunctions.isElementVisible(membershipSubMenuLink);
+		log.info("SubMenu : Membership Status Button Link is visible");
+
+		actions.moveToElement(accountAndSecurityLink).perform();
+		CommonFunctions.isElementVisible(servicesYouAreUsingSubMenuLink);
+		log.info("SubMenu : Services You Are Using Link is visible");
+
+		actions.moveToElement(accountAndSecurityLink).perform();
+		CommonFunctions.isElementVisible(orderHistorySubMenuLink);
+		log.info("Sub Menu : Order History Link is visible");
+
+	}
+
+	public void checkHeaderAndSubMenuLinksWorkingAfterLogin() throws Exception {
+
+		CommonFunctions.isElementVisible(accountAndSecurityLink);
+		CommonFunctions.waitForVisiblity(accountAndSecurityLink, waitTime);
+		accountAndSecurityLink.click();
+		log.info("Clicked on Account And Security page");
+		driver.navigate().back();
+
+		CommonFunctions.isElementVisible(membershipButton);
+		CommonFunctions.waitForVisiblity(membershipButton, waitTime);
+		membershipButton.click();
+		log.info("Clicked on Learning Window page");
+		driver.navigate().back();
+
+		clickSubMenu(accountAndSecurityLink, accountAndSecuritySubmenuLink);
```

Comment:
Error getting when executing test case 
at com.sun.proxy.$Proxy45.click(Unknown Source)
	at platform.pageobjects.Header.HeaderAfterLogin.clickSubMenu(HeaderAfterLogin.java:66)
	at platform.pageobjects.Header.HeaderAfterLogin.checkHeaderAndSubMenuLinksWorkingAfterLogin(HeaderAfterLogin.java:115)
	at platformstepdefinition.PF_HeaderAfterLogin.Validate_that_menus_and_Submenus_are_working(PF_HeaderAfterLogin.java:35)
	at ✽.Validate that menus and Submenus are working(file:///D:/Sopan/Project/Automation/PlatformTesting/platform/./src/test/java/platformfeatures/PF_HeaderAfterLogin.feature:9)

Reply:
Done changes
Please Review